geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/data/HBaseDataStoreFactory.scala

```diff
@@ -85,7 +87,7 @@ object HBaseDataStoreFactory {
   object Params {
     val BigTableNameParam  = new Param("bigtable.table.name", classOf[String], "Table name", true)
     val ConnectionParam    = new Param("connection", classOf[Connection], "Connection", false)
-    val RemoteParam        = new Param("remote.filtering", classOf[Boolean], "Remote filtering", false)
+    val RemoteParam        = new Param("remote.filtering", classOf[String], "Remote filtering", false)
```

Comment:
any reason to make this a string? Boolean should still accept a string (with conversion) and in geoserver it will present as a check-box instead of a form field.